jetcd-core/src/main/java/io/etcd/jetcd/internal/impl/LeaseImpl.java

```diff
@@ -41,16 +43,18 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.CopyOnWriteArrayList;
 import java.util.concurrent.Executors;
-import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
```

Comment:
I do prefer to use slf4j

Reply:
I actually personally do as well, but I had just copy/pasted this blindly from elsewhere [this project seems to use JUL so far](https://github.com/etcd-io/jetcd/search?q=java.util.logging&unscoped_q=java.util.logging) (see link) ... but I [now see that elsewhere the project also uses slf4j](https://github.com/etcd-io/jetcd/search?q=slf4j&unscoped_q=slf4j) - hm, this is not great. Let's first clean this up in #370, and then I change this?